backend/src/main/scala/cromwell/backend/wdl/WriteFunctions.scala

```diff
@@ -0,0 +1,54 @@
+package cromwell.backend.wdl
+
+import java.nio.file.Path
+
+import wdl4s.TsvSerializable
+import wdl4s.expression.WdlStandardLibraryFunctions
+import wdl4s.values._
+
+import scala.util.{Failure, Try}
+
+trait WriteFunctions { this: WdlStandardLibraryFunctions =>
+  import better.files._
+
+  /**
+    * Directory that will be used to write files.
+    */
+  def writeDirectory: Path
+
+  private lazy val absoluteDirectory = {
+    val abs = writeDirectory.toAbsolutePath
+    abs.createDirectories
+    abs
+  }
+
+  override def tempFilePath = absoluteDirectory.toString
+
+  def writeTempFile(path: String,prefix: String,suffix: String,content: String): String = throw new NotImplementedError("This method is not used anywhere and should be removed")
```

Comment:
Yes I wanted to but I forgot, and it's as Wdl4s method so it needs to be removed from there first.